cypress/e2e/awx/resources/workflowVisualizerCRUD.cy.ts

```diff
@@ -478,3 +459,73 @@ describe('Workflow Visualizer', () => {
     });
   });
 });
+
+describe('Workflow Visualizer: Add Nodes', () => {
+  let organization: Organization;
+  let inventory: Inventory;
+  let workflowJobTemplate: WorkflowJobTemplate;
+
+  before(function () {
+    cy.createAwxOrganization().then((org) => {
+      organization = org;
+      cy.createAwxInventory({ organization: organization.id }).then((i) => {
+        inventory = i;
+      });
+    });
+  });
+
+  beforeEach(function () {
+    cy.createAwxWorkflowJobTemplate({
+      organization: organization.id,
+      inventory: inventory.id,
+    }).then((wfjt) => (workflowJobTemplate = wfjt));
+  });
+
+  afterEach(function () {
+    cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate, { failOnStatusCode: false });
+  });
+
+  after(function () {
+    cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
+    cy.deleteAwxInventory(inventory, { failOnStatusCode: false });
```

Comment:
I believe we will need to delete the Inventory first then Org. Since the Org is associated with the inventory.